app/views/content/funding-and-support/tuition-fees.md

```diff
@@ -32,12 +32,11 @@ The maximum tuition fee for a postgraduate teacher training course for UK applic
 * £6,935 per year for a part-time course 
 
 ### Non-UK citizens
-Tuition fees for postgraduate teacher training courses for non-UK citizens vary depending on the provider. 
+Tuition fees for postgraduate teacher training courses for non-UK citizens cost:
 
-The maximum tuition fee for a postgraduate teacher training course for non-UK citizens is about:
-
-* £36,000 per year for a full-time course
-* £17,500 per year for a part-time course
+* an average of about $fees_pgitt_internationalaverage$ per year for a full-time course
```

Comment:
@Sarah-DfE I've tried to reword this section but think it needs your content expertise!